tests/integration/test_thrift/test/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ["ttypes", "constants", "TestService"]
```

Comment:
what's going on with all these test files?

Reply:
I actually...do not know. they popped up when I ran the linter but I didn't actually touch them when I was coding.

Reply:
*reset and rm from pr